data/Dockerfiles/postfix/postfix.sh

```diff
@@ -74,6 +75,48 @@ query = SELECT GROUP_CONCAT(transport SEPARATOR '') AS transport_maps
   AS transport_view;
 EOF
 
+cat <<EOF > /opt/postfix/conf/sql/mysql_recipient_dependent_default_transport_maps.cf
+user = ${DBUSER}
+password = ${DBPASS}
+hosts = mysql
+dbname = ${DBNAME}
+query = SELECT IF(EXISTS(SELECT 'relay' FROM domain
```

Comment:
@andryyy 
The problem of sender_dependent_transport_maps.cf is, that it **always** responses "smtp:" and so transport_map interprets that it should use the MX record of the domain, so I had to change it to respond "NULL" in case there is no transport_map.